### PR TITLE
 Fixes #325 : Delete confirmation issue.

### LINKED ISSUE #325: Delete confirmation issue
**Actual Behaviour**

Susi does not ask for confirmation of deleting messages.
**Expected Behaviour**

It should ask for confirmation before deletion like in whatsapp.

whatsapp image 2016-11-02 at 12 27 46 pm

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by mayank408**
Fixes issue #325
Changes: Delete confirmation added.

Screenshots for the change:

### CODE REVIEW

**Comment by mayank408** [comment]
whatsapp image 2016-11-03 at 8 25 22 pm

File: app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java [hunk 1]
```diff
@@ -586,14 +590,40 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
...
+                    d.setMessage("Are you sure ?").
+                            setCancelable(false).
+                            setPositiveButton("Delete", new DialogInterface.OnClickListener() {
+                                @Override
+                                public void onClick(DialogInterface dialog, int which) {
+
+                                    for (int i = getSelectedItems().size() - 1; i >= 0; i--) {
+                                        deleteMessage(getSelectedItems().get(i));
+                                    }
+                                    Snackbar.make(recyclerView, "Deleted Messages Successfully!!", Snackbar.LENGTH_LONG).show();
```
Comment (mayank408) [inline app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java]: @jig08 Should i change it to successfully Deleted only..

Comment (jig08) [inline app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java]: For the time being, you could put something as "Message(s)" maybe ?
Whatsapp shows a toast as "`x` messages deleted".
Any other suggestions ?

Comment (mayank408) [inline app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java]: @jig08 Ok then we can also do the same "x messages deleted".

Comment (jig08) [inline app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java]: @mayank408 That'll be great !

**Comment by mayank408** [comment]
When more than one message is deleted it shows "X messages deleted!!"
whatsapp image 2016-11-03 at 11 01 08 pm 1

When single message deleted.
whatsapp image 2016-11-03 at 11 01 08 pm

**Comment by mayank408** [comment]
@jig08 Is it all right now..

**Comment by daminisatya** [comment]
Looks great!

**Pull request merged**